.travis.yml

```diff
@@ -2,13 +2,26 @@
 sudo: required
```

Comment:
omit, and maybe add `dist:` line, like you did in the scala-xml PR, so Travis doesn't just pick one?